<?xml version="1.0" encoding="utf-8"?>
<!-- =================================================================================== -->
<!-- |    This document was autogenerated by xacro from seabotix_alpha_kinematic.xacro | -->
<!-- |    EDITING THIS FILE BY HAND IS NOT RECOMMENDED                                 | -->
<!-- =================================================================================== -->
<!-- Testing out making an xacro with the vehicle and arm together
without any uuv simulator dependencies -->
<robot name="seabotix_alpha">
  <!--
    Vehicle's parameters (remember to enter the model parameters below)
  -->
  <!--
    Visual mesh file for the vehicle, usually in DAE (Collada) format. Be sure to store the
    mesh with the origin of the mesh on the same position of the center of mass, otherwise
    the mesh pose will have to be corrected below in the <visual> block.
    Open the meshes for the RexROV vehicle in Blender to see an example on the mesh placement.
  -->
  <!-- <xacro:property name="visual_mesh_file" value="file://$(find seabotix_alpha_description)/meshes/vehicle.dae"/> -->
  <!-- Collision geometry mesh, usually in STL format (it is recommended to keep
  this geometry as simple as possible to improve the performance the physics engine
  regarding the computation of collision forces) -->
  <!-- <xacro:property name="collision_mesh_file" value="file://$(find seabotix_alpha_description)/meshes/vehicle.stl"/> -->
  <!-- We do not have different visual and collision files, as we define them directly in the URDF 
       using primitive shapes.
  -->
  <!-- Vehicle macro -->
  <!-- <xacro:macro name="seabotix_alpha_base" params="namespace *gazebo"> -->
  <link name="base_link">
    <inertial>
      <mass value="0"/>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <inertia ixx="0" ixy="0" ixz="0" iyy="0" iyz="0" izz="0"/>
    </inertial>
  </link>
  <link name="world_link_x">
    <inertial>
      <mass value="0"/>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <inertia ixx="0" ixy="0" ixz="0" iyy="0" iyz="0" izz="0"/>
    </inertial>
  </link>
  <link name="world_link_y">
    <inertial>
      <mass value="0"/>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <inertia ixx="0" ixy="0" ixz="0" iyy="0" iyz="0" izz="0"/>
    </inertial>
  </link>
  <link name="world_link_z">
    <inertial>
      <mass value="0"/>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <inertia ixx="0" ixy="0" ixz="0" iyy="0" iyz="0" izz="0"/>
    </inertial>
  </link>
  <link name="world_link_roll">
    <inertial>
      <mass value="0"/>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <inertia ixx="0" ixy="0" ixz="0" iyy="0" iyz="0" izz="0"/>
    </inertial>
  </link>
  <link name="world_link_pitch">
    <inertial>
      <mass value="0"/>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <inertia ixx="0" ixy="0" ixz="0" iyy="0" iyz="0" izz="0"/>
    </inertial>
  </link>
  <!-- Rigid body description of the base link -->
  <!-- <link name="${namespace}/base_link"> -->
  <link name="world_link_yaw">
    <!--
          Be careful to setup the coefficients for the inertial tensor,
          otherwise your model will become unstable on Gazebo
        -->
    <inertial>
      <mass value="0"/>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <inertia ixx="0" ixy="0" ixz="0" iyy="0" iyz="0" izz="0"/>
    </inertial>
    <!-- <visual>
          <origin xyz="0 0 0" rpy="0 0 0"/>
          <geometry> -->
    <!-- <mesh filename="${visual_mesh_file}" scale="1 1 1" />  -->
    <!-- <box size="1 1 1" />
            <color rgb="blue" />
          </geometry>
        </visual>

        <collision>
          <origin xyz="0 0 0" rpy="0 0 0"/>
          <geometry>
            <box size="1 1 1" />
          </geometry>
        </collision> -->
  </link>
  <material name="green_v">
    <color rgba="0.5 0.9 0.3 1"/>
  </material>
  <material name="black_v">
    <color rgba="0 0 0 1"/>
  </material>
  <material name="red_v">
    <color rgba="1 0 0 1"/>
  </material>
  <material name="blue_v">
    <color rgba="0.2 0.4 0.6 1"/>
  </material>
  <material name="grey_v">
    <color rgba=".3 .3 .3 1"/>
  </material>
  <material name="orange_v">
    <color rgba="1 .4 0 1"/>
  </material>
  <link name="vehicle_base_link">
    <visual>
      <geometry>
        <box size="0.5 0.45 0.15"/>
      </geometry>
      <material name="green_v"/>
    </visual>
  </link>
  <link name="front_bumper">
    <visual>
      <geometry>
        <cylinder length="0.45" radius="0.075"/>
      </geometry>
      <material name="green_v"/>
    </visual>
  </link>
  <joint name="fbump_link" type="fixed">
    <parent link="vehicle_base_link"/>
    <child link="front_bumper"/>
    <origin rpy="1.57579632679 0 0" xyz="0.25 0 0"/>
  </joint>
  <link name="rear_bumper">
    <visual>
      <geometry>
        <cylinder length="0.45" radius="0.075"/>
      </geometry>
      <material name="green_v"/>
    </visual>
  </link>
  <joint name="rbump_link" type="fixed">
    <parent link="vehicle_base_link"/>
    <child link="rear_bumper"/>
    <origin rpy="1.57579632679 0 0" xyz="-0.25 0 0"/>
  </joint>
  <link name="l_arrow">
    <visual>
      <geometry>
        <box size="0.05 0.318 0.01"/>
      </geometry>
      <material name="black_v"/>
    </visual>
  </link>
  <joint name="l_arrow_link" type="fixed">
    <parent link="vehicle_base_link"/>
    <child link="l_arrow"/>
    <origin rpy="0 0 0.787898163397" xyz="0.05 0.095 0.075"/>
  </joint>
  <link name="r_arrow">
    <visual>
      <geometry>
        <box size="0.05 0.318 0.01"/>
      </geometry>
      <material name="black_v"/>
    </visual>
  </link>
  <joint name="r_arrow_link" type="fixed">
    <parent link="vehicle_base_link"/>
    <child link="r_arrow"/>
    <origin rpy="0 0 -0.787898163397" xyz="0.05 -0.095 0.075"/>
  </joint>
  <link name="sf_fan">
    <visual>
      <geometry>
        <cylinder length="0.04" radius="0.06"/>
      </geometry>
      <material name="blue_v"/>
    </visual>
  </link>
  <joint name="base_to_sf_fan" type="fixed">
    <parent link="vehicle_base_link"/>
    <child link="sf_fan"/>
    <origin rpy="0 1.57579632679 0.785" xyz="0.22 -0.12 -0.14"/>
  </joint>
  <link name="sf_thruster">
    <visual>
      <geometry>
        <cylinder length="0.1" radius="0.02"/>
      </geometry>
      <material name="black_v"/>
    </visual>
  </link>
  <joint name="sf_fan_thruster_joint" type="fixed">
    <parent link="sf_fan"/>
    <child link="sf_thruster"/>
    <origin xyz="0 0 -0.06"/>
  </joint>
  <link name="pf_fan">
    <visual>
      <geometry>
        <cylinder length="0.04" radius="0.06"/>
      </geometry>
      <material name="blue_v"/>
    </visual>
  </link>
  <joint name="base_to_pf_fan" type="fixed">
    <parent link="vehicle_base_link"/>
    <child link="pf_fan"/>
    <origin rpy="0 1.57579632679 -0.785" xyz="0.22 0.12 -0.14"/>
  </joint>
  <link name="pf_thruster">
    <visual>
      <geometry>
        <cylinder length="0.1" radius="0.02"/>
      </geometry>
      <material name="black_v"/>
    </visual>
  </link>
  <joint name="pf_fan_thruster_joint" type="fixed">
    <parent link="pf_fan"/>
    <child link="pf_thruster"/>
    <origin xyz="0 0 -0.06"/>
  </joint>
  <link name="sa_fan">
    <visual>
      <geometry>
        <cylinder length="0.04" radius="0.06"/>
      </geometry>
      <material name="blue_v"/>
    </visual>
  </link>
  <joint name="base_to_sa_fan" type="fixed">
    <parent link="vehicle_base_link"/>
    <child link="sa_fan"/>
    <origin rpy="0 -1.57579632679 -0.785" xyz="-0.22 -0.12 -0.14"/>
  </joint>
  <link name="sa_thruster">
    <visual>
      <geometry>
        <cylinder length="0.1" radius="0.02"/>
      </geometry>
      <material name="black_v"/>
    </visual>
  </link>
  <joint name="sa_fan_thruster_joint" type="fixed">
    <parent link="sa_fan"/>
    <child link="sa_thruster"/>
    <origin xyz="0 0 -0.06"/>
  </joint>
  <link name="pa_fan">
    <visual>
      <geometry>
        <cylinder length="0.04" radius="0.06"/>
      </geometry>
      <material name="blue_v"/>
    </visual>
  </link>
  <joint name="base_to_pa_fan" type="fixed">
    <parent link="vehicle_base_link"/>
    <child link="pa_fan"/>
    <origin rpy="0 -1.57579632679 0.785" xyz="-0.22 0.12 -0.14"/>
  </joint>
  <link name="pa_thruster">
    <visual>
      <geometry>
        <cylinder length="0.1" radius="0.02"/>
      </geometry>
      <material name="black_v"/>
    </visual>
  </link>
  <joint name="pa_fan_thruster_joint" type="fixed">
    <parent link="pa_fan"/>
    <child link="pa_thruster"/>
    <origin xyz="0 0 -0.06"/>
  </joint>
  <link name="seabotix_bottle">
    <visual>
      <geometry>
        <cylinder length="0.26" radius="0.05"/>
      </geometry>
      <material name="black_v"/>
    </visual>
  </link>
  <joint name="sbottle_link" type="fixed">
    <parent link="vehicle_base_link"/>
    <child link="seabotix_bottle"/>
    <origin rpy="0 1.57579632679 0" xyz="-0.05 0 -0.18"/>
  </joint>
  <!-- 
      <link name="greensea">
        <visual>
          <geometry>
            <cylinder length="0.2" radius="0.04"/>
          </geometry>
          <origin rpy="0 ${pi/2} 0"/>
          <material name="grey_v"/>
        </visual>
      </link>

      <joint name="gbottle_link" type="fixed">
        <parent link="greensea"/>
        <child link="vehicle_base_link"/>
        <origin rpy="-${pi} 0 0" xyz="0 0 -0.27"/>
      </joint> -->
  <link name="dvl">
    <visual>
      <geometry>
        <cylinder length="0.25" radius="0.06"/>
      </geometry>
      <material name="grey_v"/>
      <origin rpy="0 1.57579632679 0" xyz="-0.125 0 0"/>
    </visual>
  </link>
  <joint name="dvl_link" type="fixed">
    <parent link="vehicle_base_link"/>
    <child link="dvl"/>
    <origin rpy="0 0 3.15159265359" xyz="-0.23 -0.1 -0.29"/>
  </joint>
  <link name="dvl_head">
    <visual>
      <geometry>
        <sphere radius="0.06"/>
      </geometry>
      <material name="orange_v"/>
    </visual>
  </link>
  <joint name="dvl_head_link" type="fixed">
    <parent link="dvl"/>
    <child link="dvl_head"/>
    <origin rpy="0 0 -0.787898163397" xyz="0 0 0"/>
  </joint>
  <link name="dvl_altitude"/>
  <joint name="dvl_altitude_link" type="fixed">
    <parent link="dvl_head"/>
    <child link="dvl_altitude"/>
    <origin rpy="0 1.57579632679 0" xyz="0 0 0"/>
  </joint>
  <link name="dvl_beam1">
    <visual>
      <geometry>
        <cylinder length="0.07" radius="0.015"/>
      </geometry>
      <origin rpy="0 1.57579632679 0" xyz="0.035 0 0"/>
      <material name="blue_v"/>
    </visual>
  </link>
  <joint name="dvl_beam1_link" type="fixed">
    <parent link="dvl_head"/>
    <child link="dvl_beam1"/>
    <origin rpy="0 1.05053088453 1.57579632679" xyz="0 0 0"/>
  </joint>
  <link name="dvl_beam2">
    <visual>
      <geometry>
        <cylinder length="0.07" radius="0.015"/>
      </geometry>
      <origin rpy="0 1.57579632679 0" xyz="0.035 0 0"/>
      <material name="blue_v"/>
    </visual>
  </link>
  <joint name="dvl_beam2_link" type="fixed">
    <parent link="dvl_head"/>
    <child link="dvl_beam2"/>
    <origin rpy="0 1.05053088453 -1.57579632679" xyz="0 0 0"/>
  </joint>
  <link name="dvl_beam3">
    <visual>
      <geometry>
        <cylinder length="0.07" radius="0.015"/>
      </geometry>
      <origin rpy="0 1.57579632679 0" xyz="0.035 0 0"/>
      <material name="blue_v"/>
    </visual>
  </link>
  <joint name="dvl_beam3_link" type="fixed">
    <parent link="dvl_head"/>
    <child link="dvl_beam3"/>
    <origin rpy="0 1.05053088453 0" xyz="0 0 0"/>
  </joint>
  <link name="dvl_beam4">
    <visual>
      <geometry>
        <cylinder length="0.07" radius="0.015"/>
      </geometry>
      <origin rpy="0 1.57579632679 0" xyz="0.035 0 0"/>
      <material name="blue_v"/>
    </visual>
  </link>
  <joint name="dvl_beam4_link" type="fixed">
    <parent link="dvl_head"/>
    <child link="dvl_beam4"/>
    <origin rpy="0 1.05053088453 3.15159265359" xyz="0 0 0"/>
  </joint>
  <link name="sonar">
    <visual>
      <geometry>
        <box size="0.22 0.13 0.1"/>
      </geometry>
      <material name="orange_v"/>
    </visual>
  </link>
  <joint name="sonar_link" type="fixed">
    <parent link="vehicle_base_link"/>
    <child link="sonar"/>
    <origin xyz="0.1 0.12 -0.28"/>
  </joint>
  <link name="gripper_pole">
    <visual>
      <geometry>
        <cylinder length="0.38" radius="0.015"/>
      </geometry>
      <origin rpy="0 1.57579632679 0" xyz="0.19 0 0"/>
      <material name="black_v"/>
    </visual>
  </link>
  <joint name="gripper_pole_link" type="fixed">
    <parent link="vehicle_base_link"/>
    <child link="gripper_pole"/>
    <origin rpy="0 0.1 0.3" xyz="0.15 -0.15 -0.2"/>
  </joint>
  <!-- Reach 5 Arm -->
  <link name="arm_base_link">
    <visual>
      <geometry>
        <mesh filename="package://reach_system_one_support/meshes/reach5mini/visual/Base.stl"/>
      </geometry>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <material name="black"/>
    </visual>
    <collision>
      <geometry>
        <cylinder length="0.150" radius="0.020"/>
      </geometry>
      <origin rpy="0 1.57075 0" xyz="-0.075 0 0"/>
    </collision>
    <inertial>
      <mass value="0.341"/>
      <origin xyz="-0.075 -0.006 -0.003"/>
      <inertia ixx="0.000099" ixy="0.000139" ixz="0.000115" iyy="0.002920" iyz="0.000003" izz="0.002934"/>
    </inertial>
  </link>
  <link name="shoulder_link">
    <visual>
      <geometry>
        <mesh filename="package://reach_system_one_support/meshes/reach5mini/visual/ShoulderJoint.stl"/>
      </geometry>
      <material name="black"/>
      <origin rpy="0 0 -1.57075" xyz="0 0 0"/>
    </visual>
    <collision>
      <geometry>
        <cylinder length="0.066" radius="0.020"/>
      </geometry>
      <origin rpy="0 0 0" xyz="0 0 0.010"/>
    </collision>
    <inertial>
      <mass value="0.194"/>
      <origin xyz="0.005 -0.001 0.016"/>
      <inertia ixx="0.000189" ixy="0.000005" ixz="0.000054" iyy="0.000213" iyz="0.000003" izz="0.000067"/>
    </inertial>
  </link>
  <link name="upper_arm_link">
    <visual>
      <geometry>
        <mesh filename="package://reach_system_one_support/meshes/reach5mini/visual/UpperArm.stl"/>
      </geometry>
      <origin rpy="0 -1.57075 -0.2655555" xyz="0 0 -0.075"/>
      <material name="black"/>
    </visual>
    <collision>
      <geometry>
        <cylinder length="0.1445" radius="0.020"/>
      </geometry>
      <origin rpy="0 1.57075 -0.27075" xyz="0.075 0 0"/>
    </collision>
    <collision>
      <geometry>
        <cylinder length="0.040" radius="0.020"/>
      </geometry>
      <origin rpy="0 0 0" xyz="0 0 0"/>
    </collision>
    <inertial>
      <mass value="0.429"/>
      <origin xyz="0.073 0.0 0.0"/>
      <inertia ixx="0.000087" ixy="-0.000076" ixz="-0.000010" iyy="0.003190" iyz="0.000000" izz="0.003213"/>
    </inertial>
  </link>
  <link name="forearm_link">
    <visual>
      <geometry>
        <mesh filename="package://reach_system_one_support/meshes/reach5mini/visual/ForearmJoint.stl"/>
      </geometry>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <material name="black"/>
    </visual>
    <collision>
      <geometry>
        <cylinder length="0.040" radius="0.020"/>
      </geometry>
      <origin rpy="0 0 0" xyz="0 0 0"/>
    </collision>
    <inertial>
      <mass value="0.115"/>
      <origin xyz="0.017 -0.026 -0.002"/>
      <inertia ixx="0.000120" ixy="-0.000061" ixz="-0.000001" iyy="0.000062" iyz="0.000000" izz="0.000156"/>
    </inertial>
  </link>
  <link name="wrist_link">
    <visual>
      <geometry>
        <mesh filename="package://reach_system_one_support/meshes/reach5mini/visual/Wrist.stl"/>
      </geometry>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <material name="black"/>
    </visual>
    <collision>
      <geometry>
        <cylinder length="0.140" radius="0.020"/>
      </geometry>
      <origin rpy="0 0 0" xyz="0 0 -0.070"/>
    </collision>
    <inertial>
      <mass value="0.333"/>
      <origin xyz="0.0 0.003 -0.098"/>
      <inertia ixx="0.003709" ixy="-0.000002" ixz="-0.000004" iyy="0.003734" iyz="0.000000" izz="0.000079"/>
    </inertial>
  </link>
  <link name="tip">
  </link>
  <link name="jaw1">
    <visual>
      <geometry>
        <mesh filename="package://reach_system_one_support/meshes/reach5mini/visual/Jaw1.stl"/>
      </geometry>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <material name="black"/>
    </visual>
    <collision>
      <geometry>
        <box size="0.03 0.09 0.01"/>
      </geometry>
      <origin rpy="0 0 0" xyz="-0.01 -0.045 0"/>
    </collision>
  </link>
  <link name="jaw2">
    <visual>
      <geometry>
        <mesh filename="package://reach_system_one_support/meshes/reach5mini/visual/Jaw2.stl"/>
      </geometry>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <material name="black"/>
    </visual>
    <collision>
      <geometry>
        <box size="0.03 0.09 0.01"/>
      </geometry>
      <origin rpy="0 0 0" xyz="0.01 -0.045 0"/>
    </collision>
  </link>
  <joint name="joint1" type="revolute">
    <parent link="arm_base_link"/>
    <child link="shoulder_link"/>
    <origin rpy="0 0 0" xyz="0 0 0"/>
    <!--Was <origin xyz="0 0 0" rpy="0 0 3.141592/> before-->
    <axis xyz="0 0 1"/>
    <limit effort="9.0" lower="0.0" upper="6.2" velocity="0.5"/>
  </joint>
  <joint name="joint2" type="revolute">
    <parent link="shoulder_link"/>
    <child link="upper_arm_link"/>
    <origin rpy="1.57075 1.3 0" xyz="0.020 0 0.046"/>
    <axis xyz="0 0 1"/>
    <limit effort="9.0" lower="0.0" upper="3.5" velocity="0.5"/>
  </joint>
  <joint name="joint3" type="revolute">
    <parent link="upper_arm_link"/>
    <child link="forearm_link"/>
    <origin rpy="3.1415 0 1.3" xyz="0.15 0 0"/>
    <axis xyz="0 0 1"/>
    <limit effort="9.0" lower="0.0" upper="3.5" velocity="0.5"/>
  </joint>
  <joint name="joint4" type="revolute">
    <parent link="forearm_link"/>
    <child link="wrist_link"/>
    <origin rpy="-1.57075 0 0" xyz="0.020 0 0"/>
    <axis xyz="0 0 1"/>
    <limit effort="5.0" lower="0.0" upper="6.0" velocity="1.0"/>
  </joint>
  <joint name="joint5" type="revolute">
    <parent link="wrist_link"/>
    <child link="jaw1"/>
    <origin rpy="1.57075 0 0" xyz="0.015 0 -0.140"/>
    <axis xyz="0 0 1"/>
    <limit effort="5.0" lower="0.0" upper="0.5" velocity="0.5"/>
  </joint>
  <joint name="joint6" type="revolute">
    <parent link="wrist_link"/>
    <child link="jaw2"/>
    <origin rpy="1.57075 0 0" xyz="-0.015 0 -0.140"/>
    <axis xyz="0 0 -1"/>
    <limit effort="5.0" lower="0.0" upper="0.5" velocity="0.5"/>
  </joint>
  <joint name="arm_to_tip" type="fixed">
    <parent link="wrist_link"/>
    <child link="tip"/>
    <origin rpy="0 1.57075 0" xyz="0 0 -0.180"/>
  </joint>
  <joint name="x_displacement" type="prismatic">
    <parent link="base_link"/>
    <child link="world_link_x"/>
    <origin rpy="0 0 0" xyz="0 0 0"/>
    <axis xyz="1 0 0"/>
    <limit effort="100" lower="-10" upper="10" velocity="0.5"/>
    <!-- Assume we are operating in 20x20x20m cube-->
  </joint>
  <joint name="y_displacement" type="prismatic">
    <parent link="world_link_x"/>
    <child link="world_link_y"/>
    <origin rpy="0 0 0" xyz="0 0 0"/>
    <axis xyz="0 1 0"/>
    <limit effort="100" lower="-10" upper="10" velocity="0.5"/>
    <!-- Assume we are operating in 20x20x20m cube-->
  </joint>
  <joint name="z_displacement" type="prismatic">
    <parent link="world_link_y"/>
    <child link="world_link_z"/>
    <origin rpy="0 0 0" xyz="0 0 0"/>
    <axis xyz="0 0 1"/>
    <limit effort="100" lower="-10" upper="10" velocity="0.5"/>
    <!-- Assume we are operating in 20x20x20m cube-->
  </joint>
  <joint name="roll_displacement" type="revolute">
    <parent link="world_link_z"/>
    <child link="world_link_roll"/>
    <origin rpy="0 0 0" xyz="0 0 0"/>
    <axis xyz="1 0 0"/>
    <limit effort="100" lower="-1.57579632679" upper="1.57579632679" velocity="0.315159265359"/>
    <!-- Assume we can roll by pi/2-->
  </joint>
  <joint name="pitch_displacement" type="revolute">
    <parent link="world_link_roll"/>
    <child link="world_link_pitch"/>
    <origin rpy="0 0 0" xyz="0 0 0"/>
    <axis xyz="0 1 0"/>
    <limit effort="100" lower="-1.57579632679" upper="1.57579632679" velocity="0.315159265359"/>
    <!-- Assume we can pitch by pi/2-->
  </joint>
  <joint name="yaw_displacement" type="continuous">
    <parent link="world_link_pitch"/>
    <child link="world_link_yaw"/>
    <origin rpy="0 0 0" xyz="0 0 0"/>
    <axis xyz="0 0 1"/>
    <limit effort="100" velocity="0.315159265359"/>
    <!-- Assume we have no limit on yaw-->
  </joint>
  <!-- Add a link attaching the vehicle to the world base link -->
  <joint name="joint_vehicle_base_world" type="fixed">
    <!-- <parent link="${namespace}/base_link"/> -->
    <parent link="world_link_yaw"/>
    <child link="vehicle_base_link"/>
    <origin rpy="0 0 0" xyz="0 0 0"/>
  </joint>
  <!-- Add a link attaching the arm to the world base link -->
  <joint name="joint_arm_base_world" type="fixed">
    <!-- <parent link="${namespace}/base_link"/> -->
    <parent link="world_link_yaw"/>
    <child link="arm_base_link"/>
    <origin rpy="0 0 0" xyz="0.45 0 0"/>
  </joint>
  <!-- <gazebo reference="${namespace}/base_link"> -->
  <gazebo reference="world_link_yaw">
    <selfCollide>false</selfCollide>
  </gazebo>
  <!-- Set up hydrodynamic plugin given as input parameter -->
  <!-- <xacro:insert_block name="gazebo"/> -->
  <!-- Include the thruster modules -->
  <!-- <xacro:include filename="$(find seabotix_alpha_description)/urdf/actuators.xacro"/> -->
  <!-- Include the sensor modules -->
  <!-- <xacro:include filename="$(find seabotix_alpha_description)/urdf/sensors.xacro"/> -->
  <!-- </xacro:macro> -->
</robot>
